### PR TITLE
[plugin.video.mlbtv@matrix] 2023.4.14+matrix.1

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Add-on details:

- **General**
  - Add-on name: MLB.TV®
  - Add-on ID: plugin.video.mlbtv
  - Version number: 2023.4.14+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/eracknaphobia/plugin.video.mlbtv
  
Watch every out-of-market regular season game in the office or on the go. The #1 LIVE
            Streaming Sports Service
        

### Description of changes:

            - fixed bug on stream select for games without video
            - fixed data source for game changer
            - use full team names for non-MLB teams
            - added MiLB (minor league) games
            - added country setting for better labeling international blackouts
        

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.mlbtv/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2023.3.28+matrix.1" provider-name="eracknaphobia">
+<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2023.4.14+matrix.1+matrix.1" provider-name="eracknaphobia">
```
Comment (basrieter) [inline plugin.video.mlbtv/addon.xml]: The version number is not correct.

**Review by basrieter — CHANGES_REQUESTED** [review]
Please fix the version number.

File: plugin.video.mlbtv/addon.xml (hunk 1 above)
Comment (eracknaphobia) [reply plugin.video.mlbtv/addon.xml]: sorry, i'm struggling with github actions and kodi addon submittor.

Comment (eracknaphobia) [reply plugin.video.mlbtv/addon.xml]: Should be good now.

**Review by basrieter — APPROVED** [review]

**Pull request merged**